packages: lzo: Update LZO Makefile to repair dead Main URL

nmaas87 opened the pull request:
Upgraded URL from the dead http://www.oberhumer.com/opensource/lzo/download/ to https://ftp.osuosl.org/pub/blfs/conglomeration/lzo/ (Oregon State University)

wigyori commented:
@nmaas87 - please add a "packages: lzo: " prefix, I'll merge this afterwards.

Thanks,
-w-

nmaas87 commented:
@wigyori Thank you :) I hope it is alright like this :)

wigyori commented:
Please update the commit itself, not the pull-req's cover page. :)

And the missing "Signed-off-by: ", please.

Thanks,
-w-

nmaas87 committed:
packages: lzo: Update LZO Makefile to repair dead Main URL

Upgraded URL from the dead http://www.oberhumer.com/opensource/lzo/download/ to https://ftp.osuosl.org/pub/blfs/conglomeration/lzo/ (Oregon State University)

Signed-off-by: Nico Maas <mail@nico-maas.de>

nmaas87 commented:
@wigyori Sorry for the trouble, should be correct now :)

wigyori commented:
Merged, thanks!

-w-
